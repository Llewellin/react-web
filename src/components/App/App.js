import React, { PureComponent } from 'react';

import Unibet from '../Unibet/Unibet';

export default class App extends PureComponent {
    render() {
        return (
            <div>
                <Unibet />
            </div>
        );
    }
}
