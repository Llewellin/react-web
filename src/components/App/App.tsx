import * as React from 'react';

export default class App extends React.Component {
    render() {
        return (
            <div>
                <div>Welcome to react BDD</div>
            </div>
        );
    }
}
