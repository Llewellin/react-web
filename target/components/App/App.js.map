{"version":3,"file":"App.js","sourceRoot":"","sources":["../../../src/components/App/App.tsx"],"names":[],"mappings":";;;;;;;;;;;;;;;AAAA,6BAA+B;AAE/B,yDAA0C;AAE1C;IAAiC,uBAAe;IAAhD;;IAUA,CAAC;IATU,oBAAM,GAAb;QACI,OAAO,CACH;YACI,wDAA+B;YAC/B,uCAAc;YACd,oBAAC,wBAAI,OAAG,CACN,CACT,CAAC;IACN,CAAC;IACL,UAAC;AAAD,CAAC,AAVD,CAAiC,KAAK,CAAC,SAAS,GAU/C"}