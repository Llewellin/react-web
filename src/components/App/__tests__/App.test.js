import * as React from 'react';
import {shallow} from 'enzyme';
import App from '../App';

describe('App', () => {
    const app = shallow(<App />);
    describe('render properly', () => {
        it('match snapshot', () => {
            expect(app).toMatchSnapshot();
        });
    });
});
