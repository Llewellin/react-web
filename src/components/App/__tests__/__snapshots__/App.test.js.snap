// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render properly match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App.tsx />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {},
    "ref": null,
    "rendered": null,
    "type": "App.tsx",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": "App.tsx",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
